WORKSPACE

```diff
@@ -20,3 +20,17 @@ http_archive(
     strip_prefix = "rules_pyvenv-1.2",
     url = "https://github.com/cedarai/rules_pyvenv/archive/refs/tags/v1.2.tar.gz",
 )
+
+http_archive(
+    name = "rules_nodejs",
+    sha256 = "87c6171c5be7b69538d4695d9ded29ae2626c5ed76a9adeedce37b63c73bef67",
+    strip_prefix = "rules_nodejs-6.2.0",
+    url = "https://github.com/bazelbuild/rules_nodejs/releases/download/v6.2.0/rules_nodejs-v6.2.0.tar.gz",
+)
+
+load("@rules_nodejs//nodejs:repositories.bzl", "nodejs_register_toolchains")
+
+nodejs_register_toolchains(
+    name = "nodejs",
+    node_version = "20.11.1",
+)
```

Comment:
I think you can do this all in bzlmod: `bazel_dep(name = "rules_nodejs", version = "6.2.0")`

Reply:
I need to leave a comment for this in the source code, but I think the bzlmod version of this is sort of bugged? When you do it this way it creates the `@nodejs` repository which exposes the `npm`, `npx` and other tools, which I think is preferable. When installed through bzlmod the `@nodejs` repo doesn't get created for some reason. Unless there's a good reason to prefer MODULE over WORKSPACE this seems like it might be more useful?

Reply:
I think this is the same thing but for bzlmod: https://github.com/bazel-contrib/rules_nodejs/blob/main/nodejs/extensions.bzl

Use like: 

```python
node = use_extension("@rules_nodejs//nodejs:extensions.bzl", "node")
node.toolchain(
    name = "nodejs",
    node_version = "20.11.1",
)
use_repo(node, "nodejs")
```

Then:

```
$ bazel run @nodejs//:node -- --version
INFO: Invocation ID: 40da1109-5c2a-4da7-8038-164f2f7d676a
INFO: Streaming build results to: https://app.buildbuddy.io/invocation/40da1109-5c2a-4da7-8038-164f2f7d676a
INFO: Analyzed target @@rules_nodejs~~node~nodejs//:node (0 packages loaded, 0 targets configured).
INFO: Found 1 target...
Target @@rules_nodejs~~node~nodejs_linux_amd64//:bin/node up-to-date (nothing to build)
INFO: Elapsed time: 0.194s, Critical Path: 0.07s
INFO: 1 process: 1 internal.
INFO: Build completed successfully, 1 total action
INFO: Running command line: /home/jrockway/.cache/bazel/_bazel_jrockway/7f95bd5e6dcc2e75a861133ddc7aee82/external/rules_nodejs~~node~nodejs_linux_amd64/bin/node --version
INFO: Streaming build results to: https://app.buildbuddy.io/invocation/40da1109-5c2a-4da7-8038-164f2f7d676a
v20.11.1
```

This thing also has a feature that reads .nvmrc... we should move that file from console to the top level and read it instead of putting the version in MODULE.bazel.  (Makes more sense to js developers, though I'm finally uninstalling nvm after you merge this.)